Makefile

```
@@ -205,9 +205,13 @@ refresh_product_tags:
 	${DOCKER_COMPOSE} run --rm backend perl /opt/product-opener/scripts/refresh_postgres.pl ${from}
 
 import_sample_data:
-	@echo "🥫 Importing sample data (~200 products) into MongoDB …"
-	${DOCKER_COMPOSE} run --rm backend bash /opt/product-opener/scripts/import_sample_data.sh
-
+	@ if [ "${COMPOSE_PROJECT_NAME}" = "po_off" ]; then \
```

I have a settings where I use po2 as prefix (to work in parallel on two branches), so it won't work for me.

Maybe:

```suggestion
	@ if [ "${PRODUCT_OPENER_FLAVOR_SHORT}" = "off" &&  "${PRODUCERS_PLATFORM}" != "1" ]; then \
```

---

(I tested it locally)